src/main/java/com/github/dockerjava/jaxrs/filter/ResponseStatusExceptionFilter.java

```diff
@@ -31,6 +33,8 @@
  */
 public class ResponseStatusExceptionFilter implements ClientResponseFilter {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(ResponseStatusExceptionFilter.class.getName());
```

Comment:
`LOG`

Reply:
Done

Reply:
FYI, there is 59 places using `LOG` vs 147 for `LOGGER`, should we consider changing the others for consistency (In a separate PR of course)?

Reply:
to minimise new lines i'm using shorter variant. No need to PR, not critical.

Reply:
OK